### PR TITLE
enable configuration for velero plugins

### PR DESCRIPTION
**Pull request by shubham-pampattiwar**
Add two specs `default_velero_plugins` and `custom_velero_plugins` in Velero CR spec to facilitate plugin customization and configuration.

### CODE REVIEW

File: deploy/crds/konveyor.openshift.io_v1alpha1_velero_cr.yaml [hunk 1]
```diff
@@ -4,3 +4,10 @@ metadata:
   name: example-velero
 spec:
   # Add fields here
+  default_velero_plugins:
+  - azure
+  - gcp
+  - aws
+  custom_velero_plugins:
+  - name: custom-shubham-plugin
```
Comment (dymurray) [inline deploy/crds/konveyor.openshift.io_v1alpha1_velero_cr.yaml:12]: Let's remove this for now from the default CR.

File: deploy/operator.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ spec:
       containers:
         - name: oadp-operator
           # Replace this with the built image name
-          image: "REPLACE_IMAGE"
+          image: "quay.io/spampatt/oadp-operator"
```
Comment (dymurray) [inline deploy/operator.yaml:20]: We can leave this for now. I will work to get us autobuilds in quay tied to github and we can use that image here.

File: deploy/crds/konveyor.openshift.io_v1alpha1_velero_cr.yaml (hunk 1 above)
Comment (shubham-pampattiwar) [reply deploy/crds/konveyor.openshift.io_v1alpha1_velero_cr.yaml]: Updated the code.

File: deploy/operator.yaml (hunk 2 above)
Comment (shubham-pampattiwar) [reply deploy/operator.yaml]: Done

**Pull request merged**